### ISSUE TITLE
communication approach to "Potentially out of sync" status

### ISSUE THREAD

**Issue by chaserene** [standardnotes/mobile#355]
Android, 3.5.15, GitHub APK version

I still randomly get this, and my main issue (beside the fact that my I'm indeed out of sync) is that it's not too helpful towards the user. what am I supposed to do when I see this message? should I just wait? should I close the app/force-quit/log out proper and open again? (i usually wait a few minutes, then log out proper and log in again, couldn't find anything else that works.)

is there something else (easier) that I can do to get synced?
is there something I can do to fix the root cause of the issue?
does "potentially" imply that I *may* not actually be out of sync?
this error that a critical feature of the app. would you consider displaying it somewhere where it's guaranteed to be seen, not in the hamburger sidebar?

**Comment by mobitar** [comment]
Indeed, this isn't as helpful as it can be. The goal with the first phase of this feature was essentially to first just let the user be aware that their account is not fully synced, so that if they don't see a note, for example, that they created on another device, they know that the app knows. It's really an edge case, but it does happen to people for whatever reason.

The second phase of this feature will be to allow you to act on it in more detail. On the app/desktop app you can actually already do this. On mobile this isn't yet an option.
